CONTRIBUTING.md

```diff
@@ -83,7 +83,7 @@ Check out the next section for details on how this plays out.
 
 Some Bevy Org members are also [Triage Team](https://github.com/orgs/bevyengine/teams/triage-team) members. These people can label and close issues and PRs but do not have merge rights or any special authority within the community. Existing Bevy Engine Org members can [automatically request membership](https://github.com/orgs/bevyengine/teams/triage-team/members). Once again, if you are interested don't hesitate to apply. We generally accept membership requests.
 
-We heavily limit who has merge rights within the org because this requires a large amount of trust when it comes to ethics, technical ability, and ability to enforce consistent project direction. Currently, only @cart can merge every class of change. @mockersf is allowed to merge small "uncontroversial" changes, provided they have two approvals.
+We heavily limit who has merge rights within the org because this requires a large amount of trust when it comes to ethics, technical ability, and ability to enforce consistent project direction. Currently, only @cart can merge every class of change. @mockersf is allowed to merge small "uncontroversial" changes and @alice-i-cecile is allowed to merge doc related changes, provided these changes have at least two approvals. If it came to an emergency while @cart is not around both @mockersf and @alice-i-cecile are allowed to merge changes to resolve these emergencies.
```

Comment:
> If it came to an emergency while @cart is not around both @mockersf and @alice-i-cecile are allowed to merge changes to resolve these emergencies.

I like this phrasing more: If there is an emergency that needs a quick resolution and @cart is not around, both @mockersf and @alice-i-cecile are allowed to merge changes that resolve the emergency.

Reply:
You are right, I think this phrasing is more adequate in this context.

Reply:
do I need to do somtthing for this new phrasing to be commited?

Reply:
Yes, you would have to go back into your branch that you used to create your pull request (https://github.com/bytemuck/bevy/tree/contributing_update). Inside of that branch you can do your changes to the CONTRIBUTING.md file and commit those changes to this specific branch (using `git commit`).  After that the changes will instantly appear here since this pull request compares the bevy `main` branch with your `contributing_update` branch.

You can either do this by cloning the branch to your local machine and doing the changes there (using `git clone`) or you can do the changes directly inside of GitHub. To do so you would have to go to the tab `Files changed` and press on the three small dots in the top right corner of the `CONTRIBUTING.md`. There you can select `Edit file` and do your changes. If you are done you can commit those changes.

I rewrote the sentences using the suggestions tool inside of GitHub so you can directly commit the suggestions the same way you did before using the `Commit suggestion` button.

I would suggest you to watch some basic tutorials so that you get an overview of how git works. For the most important stuff you only need a handful of `git` commands like `git clone`, `git add`, `git commit`, `git pull` and `git push`. I hope this helps!

```suggestion
We heavily limit who has merge rights within the org because this requires a large amount of trust when it comes to ethics, technical ability, and ability to enforce consistent project direction. Currently, only @cart can merge every class of change. @mockersf is allowed to merge small "uncontroversial" changes, provided these changes have at least two approvals. Same goes for @alice-i-cecile and doc related changes. If there is an emergency that needs a quick resolution and @cart is not around, both @mockersf and @alice-i-cecile are allowed to merge changes that resolve the emergency.
```

Reply:
All right, I know those basic commands and all, it was just the first "Contribution" I had ever, so I was wondering why you had the "Commit suggestion" button, but not Cart. With the little comment you post up there, I will be good to go, otherwise I'll read tutorials on the spot. Thank you very much! :)